src/js/reducers.js

```diff
@@ -645,9 +642,6 @@ function ui(state = {}, action) {
         case Actions.RESET_TIMEOUT:
             app.resetTimeout.resetTimeoutValue = action.payload.resetPeriod;
             return newState
-        case Actions.RESET_ALERT_TIMEOUT:
```

Comment:
@LiliiaShlikhtLuxoft looks like you forgot to remove `Actions.RESET_ALERT_TIMEOUT`. Or maybe it is used somewhere?

Reply:
@YuriiYefimenko it's deleted. It's in red color.

Reply:
@LiliiaShlikhtLuxoft a [constant](https://github.com/LuxoftSDL/generic_hmi/blob/3ae89b18e8553f3df1802cc93eab9b9a5a77af3f/src/js/actions.js#L31) is still there